bandpass/README.md

```diff
@@ -0,0 +1,343 @@
+
```

Comment:
Can you ask a DocPub person to review this?

Reply:
General comment: I lack the domain knowledge (signal-processing, ffts), so I can't evaluate that content beyond saying whether something seems unclear. I read for "does this seem to make sense from a using-Vertica perspective? could I follow these instructions?".  A couple of my comments might be bound up in that lack of domain knowledge, but most of them are more general I think.